graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsAutoConfiguration.kt

```diff
@@ -166,20 +168,42 @@ open class DgsAutoConfiguration(
         return Executors.newSingleThreadScheduledExecutor()
     }
 
+    @Bean
+    @ConditionalOnProperty(
+        prefix = "$AUTO_CONF_PREFIX.convertAllDataLoadersToWithContext",
+        name = ["enabled"],
+        havingValue = "true",
+        matchIfMissing = true
+    )
+    @Order(0)
+    open fun dgsWrapWithContextDataLoaderScanningInterceptor(): DgsWrapWithContextDataLoaderScanningInterceptor {
+        return DgsWrapWithContextDataLoaderScanningInterceptor()
+    }
+
+    @Bean
+    @Order(100)
```

Comment:
Mostly arbitrary. It should happen after the one that has Order(0), and the other one has Order(0) because while it should in most cases probably happen first, I don't want to unnecessarily restrict someone from putting something before it.